aether-ui/aether/ui/assets/apps/pipeline/components/DeleteModal.jsx

```diff
@@ -48,10 +48,10 @@ class DeleteModal extends Component {
     const objType = formatMessage(MESSAGES[this.props.objectType || 'pipeline'])
     const header = (
       <span>
-        <FormattedHTMLMessage
+        <FormattedMessage
```

Comment:
Checking this: https://github.com/formatjs/react-intl/blob/master/docs/Components.md#rich-text-formatting

```javascript
<FormattedMessage
  id='modal.delete.text'
  defaultMessage='Delete {objType} <b>{objName}</b>'
  values={{
    objType,
    objName: this.props.obj.name,
    b: text => <b>{text}</b> 
  }}
/>
```